app/controllers/api/v1/conversations_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class ConversationsController < Api::V1::BaseController
+      include ConversationsHelper
+
+      before_action only: %i[create index show] do
+        require_access_token %w[read]
+      end
+
+      before_action only: %i[create destroy] do
+        require_access_token %w[read write]
+      end
+
+      rescue_from ActiveRecord::RecordNotFound do
+        render json: I18n.t("api.endpoint_errors.conversations.not_found"), status: :not_found
+      end
+
+      def index
+        params.permit(:only_after, :only_unread)
+        mapped_params = {}
+        mapped_params[:only_after] = params[:only_after] if params.has_key?(:only_after)
+
+        mapped_params[:unread] = params[:only_unread] if params.has_key?(:only_unread)
+        conversations = conversation_service.all_for_user(mapped_params)
+        render json: conversations.map {|x| conversation_as_json(x) }
+      end
+
+      def show
+        conversation = conversation_service.find!(params[:id])
+        render json: conversation_as_json(conversation)
+      end
+
+      def create
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for create is too high. [27.15/20]

Reply:
FYI I am intentionally leaving these in without style check disables around the methods when it is in the controller.  My newish Ruby eyes doesn't see why these implementations are incorrect minus this warning.  I'm looking for either guidance or to hear that it's okay as is.